azure/components/db/setup.ftl

```diff
@@ -143,7 +141,57 @@
                         server.Reference
                     ]
                 /]
+                [#break]
+
+            [#case "mysql"]
+                [@createMySqlServer
+                    id=server.Id
+                    name=server.Name
+                    location=regionId
+                    createMode=createMode
+                    adminName=masterAccount
+                    adminSecret=masterSecret
+                    keyvaultId=keyVaultId
+                    skuName=sku.Processor
+                    version=engineVersion
+                    backupRetentionDays=solution.Backup.RetentionPeriod
+                    storageGB=solution.Size
+                    storageAutogrow=solution["azure:AutoGrow"]
+                /]
+
+                [#local configReferences = []]
+                [#list configs as key,value]
+                    [#local conf = solution.DBParameters[key]]
+                    [@createMySqlServerConfiguration
+                        id=value.Id
+                        name=value.Name
+                        source=key
+                        value=conf
+                        dependsOn=[server.Reference]
+                    /]
+                    [#local configReferences += [value.Reference]]
+                [/#list]
+
+                [@createMySqlServerDatabase
+                    id=db.Id
+                    name=db.Name
+                    dependsOn=[
+                        server.Reference
+                    ] + configReferences
+                /]
 
+                [#-- TODO(rossmurr4y):
+                    refactor `ignoreMissingEndpoint` alongside Service Endpoints.
+                --]
```

Comment:
```suggestion

```

Reply:
Again plagiarized from postgres. Should I remove it from there too or am I missing a difference between postgres and mysql?

Reply:
Leave it there for Postgres. 

This is a TODO tag for me to come back and perform some refactoring work.  It still has a place where it was, but it's not the sort of thing you'd copy - I would get rather confused down the line to come back to find reminders for myself that I didn't make